test/kt/esimd_radix_sort_out_of_place.cpp

```diff
@@ -44,7 +44,7 @@ test_all_view(sycl::queue q, std::size_t size, KernelParam param)
     std::cout << "\ttest_all_view(" << size << ") : " << TypeInfo().name<T>() << std::endl;
 #    endif
     std::vector<T> input(size);
-    generate_data(input.data(), size, 42);
+    TestUtils::generate_arithmetic_data(input.data(), size, 42);
```

Comment:
I would suggest making one global (in the file) definition of this constant and use that name everywhere in file below.
`constexpr int __seed = 42;`

Reply:
Let's avoid doing refactoring changes in tests which are not related to this PR. I can do it in a separate PR, though.